recipes/validictory/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "validictory" %}
+{% set version = "1.0.2" %}
+{% set sha256 = "9eeee360b9bfb6a2644796e69ae8ba0c39f04a2e75d6cd5f3707c6ac7c58d170" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - validictory
+    - validictory.tests
+
+about:
+  home: http://github.com/jamesturk/validictory
+  license: BSD
```

Comment:
The license seems to be MIT and not BSD. See https://github.com/jamesturk/validictory/blob/master/setup.py#L16